### ISSUE TITLE
Name container by default with Image alias when starting container

### ISSUE THREAD

**Issue by rhuss** [rhuss/docker-maven-plugin#48]
If an image has an `<alias>`, the container created during `docker:start` should be named with the alias. If already a container with name exists, then a suffix should be added until the name is unique. E.g. if the alias `db` is alread taken,  try `db_1`, `db_2`, .... until a free name is found.

See #35 for a discussion about this.

**Comment by jgangemi** [comment]
i have a basic implementation of this working but it requires the open pull requests for #108 to be merged first as it's built on top of that.

this is what the config looks like
```xml
<image>
  <name>busybox:latest</name>
  <alias>issue-48-named</alias>
  <namingScheme>alias</namingScheme>
  <run>
    <command>ping google.com</command>
  </run>
</image>
```
`namingScheme` is an enum that supports the following two values: `none` and `alias` w/ the default being `none`.

this resolves the use case i need to satisfy right now and the flexibility is there for more naming schemes to be added at a later date if the demand is there but for now i don't see the need.

open to suggestions if you don't like `namingScheme` as the element name and/or `none` and `alias` as the enum values.

**Comment by jgangemi** [comment]
this is ready to go pending the complete of the outstanding pull requests and any changes i need to sync w/ as a result.

i moved the `namingScheme` element into the `run` configuration where it makes more sense:

```xml
<image>
  <name>busybox:latest</name>
  <alias>issue-48-named</alias>
  <run>
    <command>ping google.com</command>
    <namingScheme>alias</namingScheme>
  </run>
</image>
```

it would be super great if `0.11.2` could be released once all the PRs (including this one when it goes up) get merged in.

**Comment by rhuss** [comment]
I'm halfway through the PRs and try my very best to release this weekend ;)

**Comment by jgangemi** [comment]
cool - i just didn't want to overwhelm you anymore then i did w/ changes stacked on changed. as soon as #110 goes in i'll sync on my end and create this PR.

**Comment by rhuss** [comment]
Fixed in 0.11.3
